### DISCUSSION TITLE
Is it possible to add drawer over drawer UI like contentful?

### DISCUSSION THREAD

**Discussion by Stupidism** [payloadcms/payload#1015] [Feature Requests & Ideas]
Now when I have a relationship inside an article. If I want to edit that related document, I need to:

1. go to that collection's list page
2. search for that specific document
3. click into the document's detail page
4. edit something and save

## Slight optimization
The easiest and most straightforward optimization I can come up with is to add a link in that relation ship card and allow editors to open the document's detail page in a new tab by a simple click

<img width="486" alt="image" src="https://user-images.githubusercontent.com/5801015/187059566-70a9059e-fda8-4ce2-99d7-8717945d9987.png">

## Heavy optimization

Like contentful, open nested drawers to edit the document

<img width="1290" alt="image" src="https://user-images.githubusercontent.com/5801015/187059491-84d2382d-193f-449f-84b4-68afe9f6b612.png">

**Comment by jmikrut** [comment]
Hey @Stupidism — we are actually going to build the heavy optimization! This is on our radar and is actually something that we've prioritized to build in sooner than later.

This is a good call. We're excited about the UX improvements that this will offer. You can expect more from us shortly.

**Reply by Stupidism** [reply]
Great to hear that!

**Reply by jmikrut** [reply]
Hey @Stupidism — this is now built!

https://github.com/payloadcms/payload/pull/1230

Gotta write tests yet, but it's on its way this week!

**Comment by mattddean** [comment]
@Stupidism while I can see that the nested modal functionality has been added to create _new_ documents, it seems like once a document is _already related_, the relationship field doesn't link to the related document. Is that right or am I missing something?

**Reply by jmikrut** [reply]
This is going to be released in like 24 hours or less ;)

**Reply by mattddean** [reply]
Oh man you're the bomb 🐐
